sponsorship_compassion/views/project_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+	<record id="view_project_intervention_form" model="ir.ui.view">
+		<field name="name">project.compassion.sponsorships</field>
+		<field name="model">compassion.project</field>
+		<field name="inherit_id" ref="child_compassion.view_compassion_project_form"/>
+		<field eval="19" name="priority"/>
+		<field name="arch" type="xml">
+			<xpath expr="//div[@name='button_box']">
+				<button name="list_sponsorships" type="object" icon="fa-home" class="oe_stat_button oe_inline">
+					<field name="open_sponsorships" widget="statinfo" string="Sponsorships"/>
```

Comment:
You corrected the wrong line! Please rebase as well and fix all travis issues...